python/ray/_raylet.pyx

```diff
@@ -1050,7 +1050,8 @@ cdef class CoreWorker:
                      PlacementGroupID placement_group_id,
                      int64_t placement_group_bundle_index,
                      c_bool placement_group_capture_child_tasks,
-                     c_string extension_data
+                     c_string extension_data,
+                     override_worker_env
```

Comment:
How about we expose `virtualenv_path` instead of this low-level option? Another question, is `PATH` guaranteed to be identical among all nodes?

Reply:
PATH is expected to be identical. We have to drop down to this level because for Python, virtualenv_path isn't the only variable of concern here. There are various variable depend on the interpreter and libraries:
- PATH
- LD_PRELOAD_PATH
- LD_LIBRARY_PATH
- CONDA_ENV
- CONDA_PREFIX
- VIRTUAL_ENV

The most important one of just PATH which is not constrained to be just Python.